_assets/styles/global.scss

```diff
@@ -55,13 +58,14 @@ abbr {
 a {
   color: $black;
   text-decoration: none;
-  -webkit-transition: background-color, border-color, color;
+  -webkit-transition: background-color, border-color, color, fill;
```

Comment:
no need for the browser prefixes, postcss/autoprefixer has got your back! 😎

Reply:
These were already here. So I left 'em.

Reply:
ah, I see

Reply:
right, indeed, got loads of browser prefixes hanging around - let me pick this up under a new PR